src/tree/AttachedAccountsTreeItem.ts

```diff
@@ -299,6 +319,10 @@ export class AttachedAccountsTreeItem extends AzureParentTreeItem {
 
             label = label || `${id} (${getExperienceFromApi(api).shortName})`;
             treeItem = new MongoAccountTreeItem(this, id, label, connectionString, isEmulator);
+            // tslint:disable-next-line: possible-timing-attack
```

Comment:
Lol this sounds scary I've never seen it before. Why do we need to disable the rule here?

Reply:
Lol I know, the lint rule has to do with passwords in general more related to security issues. I am not sure why it shows up here tho, even the earlier version of this code disabled it. The same comment is present again few lines above for mongo check.